crux-mir/lib/dlmalloc/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "dlmalloc"
+version = "0.2.3"
```

Comment:
Is this exactly the same as the upstream version of `dlmalloc-0.2.3`, or did you have to apply any patches to make it work with `mir-json`/`crux-mir`? If you did have to patch it, it would be worth putting a high-level description of the changes in `crux-mir/lib/Patches.md`.

Reply:
Yes it is exactly the same, no patches are required.